tests/access_tests.py

```diff
@@ -236,10 +236,24 @@ def create_access_request(ds_type, ds_name, role_name):
             'datasource_access', druid_ds_2_perm)
         self.assertIn(perm_view, druid_role.permissions)
 
+        # Case 5. Create new access request to table
+        # Grant user alpha access
+        # Check if gamma access request was deleted
+
+        # create request on energy usage
+        access_request5 = create_access_request('table', 'energy_usage', TEST_ROLE_NAME)
+        ds_5_id = access_request5.datasource_id
+        # grant Admin role
+        self.get_resp(GRANT_ROLE_REQUEST.format('table', ds_5_id, 'gamma', 'Admin'))
```

Comment:
this test case doesn't cover the cleanup. 
please add 2 test cases:
1.  2 DARs, 2 users, datasource is added to the role for 1 user and that resolves it for another
2. 2DARs, 2 users, user 1 became alpha, user 2 DAR approved and user 1 DAR cleaned up